test/controllers/aip/v1/aip_items_contoller_test.rb

```diff
@@ -0,0 +1,156 @@
+require 'test_helper'
+require 'json-schema'
+require Rails.root.join('test/support/aip_helper')
```

Comment:
shouldn't need the `Rails.root.join` here? Should just be able to just use `require 'test/support/aip_helper'`?

```suggestion
require 'test/support/aip_helper'
```

Reply:
I tried that but I get the error ``LoadError: cannot load such file -- test/support/aip_helper``
Maybe I need to add it to a path somewhere ?

Reply:
This works:
```
require_relative '../../../support/aip_helper.rb'
```
Don't know if its better than the original though since it tells me where the file actually is relative to the project:
```
require Rails.root.join('test/support/aip_helper')
```

Reply:
We can leave it as is 👍